src/components/Blockquote/Blockquote.story.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Did you mean to switch from stories.mdx? This is the preferred pattern since Storybook v6.0, which we upgraded to last week

Reply:
Oh, I was just switching to story.jsx because that's what we use everywhere else. So it's stories.mdx with no story.jsx file moving forward?

Reply:
I'll update the readme if that's the case

Reply:
That's right. Thanks